docs/reference/slm/slm-retention.asciidoc

```diff
@@ -11,15 +11,15 @@ run and for how long, the second configured on a policy for which snapshots shou
 retention.
 
 The cluster level settings for retention are shown below, and can be changed dynamically using the
-<<cluster-update-settings,cluster-update-settings>> API:
+<<cluster-update-settings>>:
```

Comment:
I'd suggest reverting this. Here's how it renders in the preview: 
<img width="738" alt="Screen Shot 2019-11-26 at 11 22 46 AM" src="https://user-images.githubusercontent.com/40268737/69652141-30724280-103f-11ea-88de-d64607923719.png">

http://elasticsearch_48705.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/slm-retention.html